sdk-extensions/autoconfigure/README.md

```diff
@@ -159,6 +159,16 @@ The logging exporter prints the name of the span along with its attributes to st
 | otel.metrics.exporter=logging | OTEL_METRICS_EXPORTER=logging | Select the logging exporter for metrics                              |
 | otel.logs.exporter=logging    | OTEL_LOGS_EXPORTER=logging    | Select the logging exporter for logs                                 |
 
+### Logging OTLP JSON exporter
+
+The logging-otlp exporter writes the telemetry data to the JUL logger in OLTP JSON form. It's a more verbose output mainly used for testing and debugging.
+
+| System property                    | Environment variable               | Description                                        |
+|------------------------------------|------------------------------------|----------------------------------------------------|
+| otel.traces.exporter=logging-otlp  | OTEL_TRACES_EXPORTER=logging-otlp  | Select the logging OTLP JSON exporter for tracing  |
+| otel.metrics.exporter=logging-otlp | OTEL_METRICS_EXPORTER=logging-otlp | Select the logging OTLP JSON exporter for metrics  |
+| otel.logs.exporter=logging-otlp    | OTEL_LOGS_EXPORTER=logging-otlp    | Select the logging OTLP JSON exporter for logs     |
```

Comment:
are these officially specced properties?

Reply:
Sorry for the naive question, but what makes properties "officially specced"?

Reply:
The opentelemetry-specification [defines](https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/sdk-environment-variables.md#exporter-selection) defines the set known exporter name values. `logging-otlp` is not one because the idea of an exporter that logs to the console in OTLP JSON format is not mentioned in the spec, and may be unique to the java SDK.

Reply:
I think its probably fine to add this:
1. Autoconfigure is still experimental.
2. If we follow through with [this plan](https://github.com/open-telemetry/opentelemetry-java/pull/4879/files#r1015931599) to implement as a provider in the `:exporters:logging-otlp` module, then it's less relevant that this isn't defined in the spec.
3. It seems low probably that the spec defines the OTLP logging exporter concept anytime soon. We obviously think this is a useful concept and have even stabilized the exporters. Seems wrong to withhold this value from autoconfigure users indefinitely while we wait for the spec to define this.

Reply:
Just wondered if we should tack the `experimental` into the env vars/sys props like we have in the past for pre-specced items.

Reply:
I thought about that a bit. Its a new pattern because the `experimental` would land in the value, rather than the key. I.e. we haven't don't anything like `otel.traces.exporter=experimental-logging-otlp` before.

Don't feel strongly either way.